script.js

```diff
@@ -0,0 +1,90 @@
+/* Julie Rizzino
+script.js
+INFO 2134
+Thoendel
+3/20/20 */
+
+const form = document.getElementById('contactForm');
+//label the element id contactForm as the constant form
+const meal = document.getElementById('meal');
+//label the element id meal as the constant meal
+const mealOptions = document.getElementById('mealOptionsHolder');
+//label the element id mealOptionsHolder as the constant mealOptions
+const breakfast = document.getElementByValue('breakfast');
+//label the element value breakfast as the constant breakfast
+const breakfastOptions = document.getElementById('breakfastOptions');
+//label the element id breakfastOptions as the constant breakfastOptions
+const lunch = document.getElementByValue('lunch');
+//label the element value lunch as the constant lunch 
+const lunchOptions = document.getElementById('lunchOptions');
+//label the element id lunchOptions as the constant lunchOptions
+const dinner = document.getElementByValue('dinner');
+//label the element value dinner as the constant dinner
+const dinnerOptions = document.getElementById('dinnerOptions');
+//label the element id dinnerOptions as the constant dinnerOptions
+
+const firstName = form.firstName;
+//label the constant firstName from the constant form
+const lastName = form.lastName;
+//label the constant lastName from the constant form
+
+const age = document.getElementById('age');
+//label the element id age as the constant age
+
+meal.addEventListener('select', validate, false);
+//adding event listener to wait for meal option to be selected
+function checkMeal(event){
+    if(meal == breakfast){
+        mealOptions = breakfastOptions;
+        mealOptions.breakfastOptions.style.display = 'block';
+    }else if(meal == lunch){
+        mealOptions = lunchOptions;
+        mealOptions.lunchOptions.style.display = 'block';
+    }else if(meal == dinner){
+        mealOptions = dinnerOptions;
+        mealOptions.dinnerOptions.style.display = 'block';
+    }
+    }
+    //if then else function created to check the meal picked and display the meal options for the selected meal
+
+firstName.addEventListener('blur', validate, false);
+//adding event listener to wait for the firstName field to lose focus
+function validateInline1(event){
```

Comment:
If you are going to define this as a function - do it outside the listener so that you can define one function to handle any string (first name or lastname)